src/components/NcSelect/NcSelect.vue

```diff
@@ -1194,15 +1195,19 @@ body {
 
 	.vs__search {
 		text-overflow: ellipsis;
+		color: var(--color-main-text);
+
+		&::placeholder {
+			color: var(--color-text-maxcontrast);
+		}
 	}
 
 	.vs__search, .vs__search:focus {
-		margin: 2px 0 0;
+		margin: 0;
 	}
 
 	.vs__dropdown-toggle {
 		position: relative;
-		max-height: 100px;
```

Comment:
Why this change? This can lead to really long element instead of scrolling. (like if you select many tags or something)

Reply:
Ah ok

Reply:
This is changing the behavior a lot:

before | after
---|---
![Screenshot 2024-07-22 at 15-25-07 Nextcloud Vue Style Guide](https://github.com/user-attachments/assets/ba5a4278-475f-4c78-8add-588dcb9b14e9)|![Screenshot 2024-07-22 at 15-24-58 Nextcloud Vue Style Guide](https://github.com/user-attachments/assets/e87c69cf-1e6b-4d85-ad51-3efdbaece6d4)